reference/docs-conceptual/learn/experimental-features.md

```diff
@@ -88,6 +89,19 @@ In this example, a job is started and a breakpoint is set to break when the `Set
 run. The runspace is stored in a variable and passed to the `Get-PSBreakPoint` command with the
 **Runspace** parameter. You can then inspect the breakpoint in the `$breakpoint` variable.
 
+## PSAMSIMethodInvocationLogging
+
+The Windows Antimalware Scan Interface (AMSI) is an API that allows application actions to be passed
+to an antimalware scanner, such as Windows Defender, to be scanned for malicious payloads. Beginning
```

Comment:
```suggestion
The Windows Antimalware Scan Interface (AMSI) is an API that allows applications to pass actions 
to an antimalware scanner, such as Windows Defender, for detecting malicious payloads. Beginning
```

Just a quick pass at removing some passive voice and clarifying value for this feature.